index.js

```diff
@@ -98,7 +98,7 @@ class SitemapStream extends EventEmitter {
 
     const isWritten = this.writer.write(`${loc}${lastMod}${changeFreq}${priority}${mobile}`);
 
-    if (isWritten) this.nbInjectedUrls++;
+    this.nbInjectedUrls++;
```

Comment:
Please note that `entry` would be still written using a current `writer` even when `isWritten === false`, i.e. operation was buffered, but will eventually hit the underlying resource.